aten/src/ATen/native/mkldnn/xpu/qconv_pt2e.cpp

```diff
@@ -0,0 +1,116 @@
+#include <c10/core/MemoryFormat.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/native/mkldnn/xpu/detail/oneDNN.h>
+#include <torch/library.h>
+
+#include <iostream>
+
+using namespace at::native::onednn;
+namespace at::native::xpu {
+
+at::Tensor qconv_prepack_xpu(
+    at::Tensor weight,
+    at::Tensor weight_scales,
+    double input_scale,
+    int64_t input_zero_point,
+    torch::List<int64_t> stride,
+    torch::List<int64_t> padding,
+    torch::List<int64_t> dilation,
+    int64_t groups,
+    std::optional<torch::List<int64_t>> input_shape
+){
```

Comment:
Pls. update the style.

Reply:
thanks for your suggestions, I have update the coding style of this file